environment.conf.json

```diff
@@ -1,4 +1,12 @@
 {
+  "meta": {
+    "androidIcon": "/icon_192x192.png",
+    "backgroundColor": "#ffffff",
+    "favicon": "/favicon.png",
+    "iosIcon": "/icon_180x180.png",
+    "statusBarStyle": "default",
+    "themeColor": "#ec0000"
+  },
```

Comment:
Since this is all manifest data, I would consider renaming `meta` to `manifest`.

Reply:
It is not, though. `favicon` goes into a `meta` tag and so does `iosIcon`.

Reply:
Okay, not all props are meta tag values, but some are and some are not. So, still, the `meta` parent prop doesn't seem like a good container name.

Reply:
That's right. Meta, as in the general meaning of the word, would be fitting, but since the `meta` tag in the head is called the same it's confusing. I'll see if I can think of another name.

Reply:
How about `head`, as in the `head` tag? Or `htmlHead`?